packages/primitives/collapsible/src/collapsible-root.directive.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    AfterViewInit,
+    ContentChild,
+    Directive,
+    EventEmitter,
+    inject,
+    InjectionToken,
+    Input,
+    Output
+} from '@angular/core';
+import { asyncScheduler } from 'rxjs';
+
+import { RdxCollapsibleContentDirective } from './collapsible-content.directive';
+
+const RdxCollapsibleToken = new InjectionToken<RdxCollapsibleRootDirective>('RdxCollapsibleToken');
+
+export function injectCollapsible(): RdxCollapsibleRootDirective {
+    return inject(RdxCollapsibleRootDirective);
+}
+
+export type RdxCollapsibleState = 'open' | 'closed';
+
+@Directive({
+    selector: '[CollapsibleRoot]',
+    standalone: true,
+    exportAs: 'collapsibleRoot',
+    providers: [{ provide: RdxCollapsibleToken, useExisting: RdxCollapsibleRootDirective }],
+    host: {
+        '[attr.data-state]': 'getState()'
+    }
+})
+export class RdxCollapsibleRootDirective implements AfterViewInit {
+    private _open = false;
+    @Input() disabled = false;
+    @Input() set open(value: boolean) {
+        if (value !== this._open) {
+            this.openChange.emit(value);
+        }
+
+        this._open = value;
+    }
+    @Output() openChange = new EventEmitter<boolean>();
+    @ContentChild(RdxCollapsibleContentDirective) contentDirective?: RdxCollapsibleContentDirective;
```

Comment:
you can use `contentChild` with InjectToken from ContentDirective

Reply:
Done!